pkg/operator/controllers/guardrails/gktemplates/aro-deny-labels.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: templates.gatekeeper.sh/v1
+kind: ConstraintTemplate
+metadata:
+  name: arodenylabels
+  annotations:
+    description: >-
+      Prevents changes in resources that contain specified labels, with values
```

Comment:
very nice implementation! and nice to see `opa` will take place to establish a policy architecture overall, here's a small question: would we export this to an external repo that both osd/rosa and aro can call? Because it looks usable for osd/rosa as well

Reply:
AFAIK, osd/rosa has customised webhook implementation of similar features, so there is interests to migrate to gatekeeper? we sure can have some discussion if it is.

Reply:
From my point of view, yeah to be able to manage the policies at hand would bring us the flexibility to sync configurations for all the fleets without considering the underlying infrastructure and make sure they are in the same state in terms of policy governance

A security patch can be applied to all fleets instantly including aro-osd-rosa which would be a lot faster than trying to implement it in 2 different ways 

Although webhooks have their own advantages (like they are native, using go etc) it might be nice to have a second standard option amongst aro-rosa-osd etc

Reply:
there is a plan to move away from the customized webhooks for OSD as far I know, PMs and such wanted to converge and use OPA